src/core/service.c

```diff
@@ -1723,27 +1723,31 @@ static int service_spawn_internal(
                  * in ENOTCONN), and just use whate we can use. */
 
                 if (getpeername(s->socket_fd, &sa.sa, &salen) >= 0 &&
-                    IN_SET(sa.sa.sa_family, AF_INET, AF_INET6, AF_VSOCK)) {
+                    IN_SET(sa.sa.sa_family, AF_INET, AF_INET6, AF_VSOCK, AF_UNIX)) {
                         _cleanup_free_ char *addr = NULL;
                         char *t;
                         unsigned port;
```

Comment:
This can be moved into the if branch below now

Reply:
Hmm, don't really understand what could be changed here, could you clarify?

Reply:
That means, `port` can be declared in the `if` block below.

Reply:
Oh right, I was scratching my head looking at the diff :smile:. Fixed.